deal.II-toolchain/packages/scalapack.package

```diff
@@ -0,0 +1,86 @@
+################################################################################
+## ScaLAPACK                                                                  ##
+################################################################################
+
+VERSION=2.1.0
+EXTRACTSTO=scalapack-${VERSION}
+
+NAME=v${VERSION}
+PACKING=.tar.gz
+
+SOURCE=https://github.com/Reference-ScaLAPACK/scalapack/archive/
+CHECKSUM=e9ae970238030fa1e8ee022a9d3e6fbb
+
+BUILDCHAIN=cmake
+
+BUILDDIR=${BUILD_PATH}/scalapack-${VERSION}
+INSTALL_PATH=${INSTALL_PATH}/scalapack-${VERSION}
+
+
+
+# Set compilers & compiler options
+if [ ! -z "${CC}" ]; then
+    CONFOPTS="${CONFOPTS} \
+      -D CMAKE_C_COMPILER=${CC}"
+fi
+
+if [ ! -z "${CXX}" ]; then
+    CONFOPTS="${CONFOPTS} \
+      -D CMAKE_CXX_COMPILER=${CXX}"
+fi
+
+if [ ! -z "${FC}" ]; then
+    CONFOPTS="${CONFOPTS} \
+      -D CMAKE_Fortran_COMPILER=${FC}"
+fi
+
+# Set Fortran flags
+FORTRAN_MAJOR_VERSION=$(echo __GNUC__ | ${FC} -E -xc - | tail -n 1)
```

Comment:
this works only if FC is given

Reply:
I will add a test if FC is given.

But what can we do, when FC is not given? We still need  to check the compiler version, since we need to set the Fortran_FLAGS correspondingly.

Reply:
Well, than you can't do much, you can try finding `gfortran` as it is done in `candi.sh` for tools like `wget`.
But if `FC` is however not set, this may end in a strange error

Reply:
simply put `if [[ -n ${FC} ]]` around this statement, this should be good

Reply:
I added a test if `FC` is given.